MCDTool.egg-info/PKG-INFO

```diff
@@ -0,0 +1,14 @@
+Metadata-Version: 2.1
+Name: MCDTool
+Version: 0.0.1
+Summary: A probabilistic Multi Criteria Decision Analysis
+Home-page: UNKNOWN
+Author: Flaminia Catalli - wetransform GmbH & Matteo Spada - Zurich University of Applied Sciences
+Author-email: fc@wetransform.to, spaa@zhaw.ch
+License: UNKNOWN
```

Comment:
We should add a License here. The EPL should be OK

Reply:
I would suggest we use one of the following licenses:
- LGPL 3.0 (Which focuses on contributions from a community, would be the same that we use for our other open source tools)
- EUPL + Commons Clause (which is a copyleft license, but with the commons clause it requires dual licensing for offering it as a commercial service)

Reply:
We initially agreed on the EPL v 2.0 license. Now is displayed also in the description of the package.